Source/ASTextNode2.mm

```diff
@@ -315,6 +338,98 @@ - (UIAccessibilityTraits)defaultAccessibilityTraits
   return UIAccessibilityTraitStaticText;
 }
 
+/// Uses the given layout, node and container node to calculate the accessibiliyty frame for the given ASTextNodeAccessiblityElement in screen coordinates.
+static void ASUpdateAccessibilityFrame(ASTextNodeAccessiblityElement *accessibilityElement, ASTextLayout *layout, ASDisplayNode * _Nullable containerNode, ASDisplayNode *node) {
+  // This needs to be in the first non layer nodes coordinates space
+  containerNode = containerNode ?:  node.firstNonLayerNode;
+  CGRect textLayoutFrame = CGRectZero;
+  NSRange accessibilityRange = accessibilityElement.accessibilityRange;
+  if (accessibilityRange.location == NSNotFound) {
+    // If no accessibilityRange was specified (as is done for the text element), just use the
+    // label's range and clampt to the visible range otherwise the returned rect would be invalid.
+    NSRange range = NSMakeRange(0, accessibilityElement.accessibilityLabel.length);
+    range = NSIntersectionRange(range, layout.visibleRange);
+    textLayoutFrame = [layout rectForRange:[ASTextRange rangeWithRange:range]];
+  } else {
+    textLayoutFrame = [layout rectForRange:[ASTextRange rangeWithRange:accessibilityRange]];
+  }
+  CGRect accessibilityFrame = [node convertRect:textLayoutFrame toNode:containerNode];
+  accessibilityElement.accessibilityFrame = UIAccessibilityConvertFrameToScreenCoordinates(accessibilityFrame, containerNode.view);
+}
+
+- (BOOL)isAccessibilityElement
+{
+  if (!ASActivateExperimentalFeature(ASExperimentalTextNode2A11YContainer)) {
+    return [super isAccessibilityElement];
+  }
+
+  // If the ASTextNode2 should act as an UIAccessibilityContainer it has to return
+  // NO for isAccessibilityElement
+  return NO;
+}
+
+- (NSInteger)accessibilityElementCount
+{
+  if (!ASActivateExperimentalFeature(ASExperimentalTextNode2A11YContainer)) {
+    return [super accessibilityElementCount];
+  }
+
+  return self.accessibilityElements.count;
+}
+
+- (NSArray *)accessibilityElements
+{
+  if (!ASActivateExperimentalFeature(ASExperimentalTextNode2A11YContainer)) {
+    return [super accessibilityElements];
+  }
+
+  NSAttributedString *attributedText = _attributedText;
```

Comment:
Use ivar directly. There's no overhead to strong/assign ivar accesses.

Reply:
Done